scripts/sct_compute_snr.py

```diff
@@ -10,28 +10,31 @@
 # Authors: Simon LEVY
 #
 # About the license: see the file LICENSE.TXT
-#########################################################################################
-
+########################################################################################
 
 from __future__ import division, absolute_import
 
 import sys
-import operator
 import numpy as np
 from msct_parser import Parser
-from spinalcordtoolbox.image import Image
+from spinalcordtoolbox.image import Image, empty_like
 from spinalcordtoolbox.utils import parse_num_list
-from spinalcordtoolbox.template import get_slices_from_vertebral_levels
 import sct_utils as sct
 
 
+# PARAMETERS
+class Param:
+    # The constructor
+    def __init__(self):
+        self.almost_zero = 0.000000001
```

Comment:
There is also a numpy machine epsilon for something similar:

    import numpy as np
    print(np.finfo(float).eps)

Reply:
thanks for the tip! will do the modif